Workflows/Operation/01_ManageWorkflows.md

```diff
@@ -0,0 +1,313 @@
+  A window to search for an action is displayed.
+
+  ![Search action](/Assets/Screenshots/Workflows/Workflows/SearchAction.png "[Search action]")
+
+4. Click the *search for an action* field and enter the name of the action or a keyword you are searching for.
+
+  > [Info] The list of actions is filtered for your keyword as you type.
+
+5. Click the action you want to add in the list of actions.    
+  The action is added to the workflow and connected to the selected place.
+
+  ![Add action](/Assets/Screenshots/Workflows/Workflows/AddAction.png "[Add action]")
+
+
+#### Edit an action
+
+1. Click the action you want to edit.  
+  The action is highlighted and its settings are displayed in the settings side bar on the right.
+
+  ![Action](/Assets/Screenshots/Workflows/Workflows/Action.png "[Action]")
+
+  > [Info] The settings displayed in the settings side bar may differ depending on the respective action. The sections *Configuration* and *Static Inputs* are only available for certain actions.     
+
+
+2. Edit the settings field in the settings side bar. You can change the label, the queue type, the task event, the configuration and the static inputs as described below:
+
+  + Click the *Label* field and edit the name of the action displayed in the workflow editor.
+
+  + Click the *Queue Type* drop-down list and select the appropriate queue type for the action. All available queue types are displayed in the drop-down list. By default the *Default* queue type is selected. You can define a different queue type for each action within the workflow.
+
+  + Click the *Task Event* drop-down list and select an event connected to the action. The event is triggered and the corresponding task is assigned when the action is failed within a process. All available task events are displayed in the drop-down list. By default, the *ActindoWorkflow Process Execution Failed* event is preselected.
+
+  + Edit the field(s) in the *Configuration* section to define further configuration settings to the selected action. This section is only displayed for certain actions. The fields displayed in the *Configuration* section depend on the selected action.
+
+  + Click the ![Add](/Assets/Icons/Plus04.png "[Add]") (Add) button right to an input name in the *Static Inputs* section to add a static data value to the corresponding input. All input ports that are not yet connected to a place are displayed in this section. Enter the appropriate value in the text field below the input name. The value entered must be a valid JSON value. Click the ![Delete](/Assets/Icons/Trash07.png "[Delete]") (Delete) button right to a value to delete it. This section is only displayed for certain actions.
+
+  > [Info] Input ports with an static input value cannot be connected to a place. Therefore, an arc without a place is displayed in front of an input port with a static input value. Delete the static input value from the port to be able to connect it with a place.     
+
```

Comment:
Is the little arrow an arc or simply an arrow?

Reply:
good question!